docs-www/gatsby-config.js

```diff
@@ -23,8 +23,23 @@ module.exports = {
   // URL prefix on production environment. For more info see https://www.gatsbyjs.org/docs/path-prefix/
   pathPrefix: process.env.PATH_PREFIX || null,
 
-  plugins: pluginConfigFactory({
-    config: `${__dirname}/gatsby-docs-kit.yml`,
-    resources: path.resolve(__dirname, '../docs'),
-  }),
+  plugins: [
+    ...pluginConfigFactory({
+      config: `${__dirname}/gatsby-docs-kit.yml`,
+      resources: path.resolve(__dirname, '../docs'),
+    }),
+    {
+      resolve: `gatsby-plugin-google-analytics`,
+      options: {
+        trackingId: 'UA-62818184-6',
+        head: false,
+        anonymize: true,
+        respectDNT: true,
+        pageTransitionDelay: 0,
+        sampleRate: 5,
+        siteSpeedSampleRate: 10,
+        cookieDomain: 'brainhubeu.github.io',
```

Comment:
Why do we track only 5% for sampleRate and 10% of users for siteSpeedSampleRate?

Reply:
I thought these options from the docs would be good but I can remove them

Reply:
I removed them in https://github.com/brainhubeu/react-carousel/pull/342/commits/c91a5ce71cb9b9b59f5a65c168db392f46ba8b4d

Reply:
I'm just wondering what values would be optimal for our needs. According to [docs](https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#create) the default value for Sample Rate is `100` which means 100% of users will be tracked. And for the `siteSpeedSampleRate` the default value is 10% of users